node/tessel.js

```diff
@@ -337,17 +337,35 @@ function SPI(params, port) {
 
     /* spi baud rate is set by the following equation:
     *  f_baud = f_ref/(2*(baud_reg+1))
-    *  max baud rate is 24MHz for the SAMD21
+    *  max baud rate is 24MHz for the SAMD21, min baud rate is 93750 without a clock divisor
+    *  with a max clock divisor of 255, slowest clock is 368Hz unless we switch from 48MHz xtal to 32KHz xtal
     */
     // default is 2MHz
     params.clockSpeed = params.clockSpeed ? params.clockSpeed : 2e6;
 
-    if (params.clockSpeed > 24e6 || params.clockSpeed < 93750) {
-        throw new Error('SPI Clock needs to be between 24e6 and 93750Hz.');
+    // if speed is slower than 93750 then we need a clock divisor
+    if (params.clockSpeed > 24e6 || params.clockSpeed < 368) {
+        throw new Error('SPI Clock needs to be between 24e6 and 368Hz.');
     }
 
     this.clockReg = parseInt(48e6/(2*params.clockSpeed) - 1);
 
+    // find the smallest clock divider such that clockReg is <=255
+    if (this.clockReg > 255) {
+        this._clockDiv = parseInt(48e6/(params.clockSpeed*(2*255+2)));
```

Comment:
Consider `Math.floor` or `Math.round` for float->int conversion, instead of going float->string->int with parseInt.